src/iarc7_motion/iarc_tasks/task_commands.py

```diff
@@ -5,7 +5,7 @@
 from geometry_msgs.msg import TwistStamped
 
 class VelocityCommand(object):
-    def __init__(self,  target_twist=None):
```

Comment:
Yes I checked too. I removed it so that it couldn't be used as a fake nop command anymore. You have to construct a twiststamped which would be unproductive.